src/org/authInfo.ts

```diff
@@ -372,14 +372,16 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
    * ```
    * @param sfdxAuthUrl
    */
-  public static parseSfdxAuthUrl(sfdxAuthUrl: string) {
+  public static parseSfdxAuthUrl(
+    sfdxAuthUrl: string
+  ): Pick<AuthFields, 'clientId' | 'clientSecret' | 'refreshToken' | 'loginUrl'> {
     const match = sfdxAuthUrl.match(
       /^force:\/\/([a-zA-Z0-9._-]+):([a-zA-Z0-9._-]*):([a-zA-Z0-9._-]+={0,2})@([a-zA-Z0-9._-]+)/
     );
 
     if (!match) {
       throw new SfdxError(
-        'Invalid sfdx auth url. Must be in the format `force://<clientId>:<clientSecret>:<refreshToken>@<loginUrl>`. The instanceUrl must not have the protocol set.',
+        'Invalid SFDX auth URL. Must be in the format "force://<clientId>:<clientSecret>:<refreshToken>@<instanceUrl>".  Note that the SFDX auth URL uses the "force" protocol, and not "http" or "https".  Also note that the "instanceUrl" inside the SFDX auth URL doesn\'t include the protocol ("https://").',
```

Comment:
@mdonnalley should we start considering blurring the line for messages considering these changes will also be used in sf?

Reply:
This particular method wouldn't surface in sf since it's only for authing with an sfdx auth url

But, yes, we should probably give more thought to making messages applicable to both executables. I think that's outside the scope of this PR though